banking-app/test/run.md

```diff
@@ -0,0 +1,63 @@
+# CCF Application Test Environment Setup
+
+## Azure Hosting Environment
+- Azure Confidential Computing VM with [TEE hardware [Azure DC series] (Confidential Linux Image)](https://learn.microsoft.com/en-us/azure/confidential-computing/quick-create-portal)
+    - SGX hardware is installed as TEE (Trusted Execution Environment) 
+    - Both Release and Virtual modes will be supported
+- Or Normal Azure Virtual Machine [(Confidential Linux Image)](https://learn.microsoft.com/en-us/azure/confidential-computing/create-confidential-vm-from-compute-gallery)
+    - No TEE hardware installed 
+    - Only Virtual mode will be supported
+
+## Remote Access your Azure VM using SSH
+```bash
+sudo ssh -i ./key.pem user@VMIP
+```
+## Install CCF runtime on fresh VM
+```bash
+./scripts/setup_ccf_on_vm.sh
```

Comment:
Is this `setup_ccf_on_vm.sh` script supposed to be in this PR ? I don't see it.

Reply:
The main PR focus is revision of GET_STARTED.md, other parts is just trials.